backend/curation/api/curation-api.yml

```diff
@@ -511,34 +538,141 @@ paths:
         "401":
           $ref: "#/components/responses/401"
 
-  /v1/collections/{collection_id}/datasets/{dataset_id}/assets:
+#  /v1/collections/{collection_id}/datasets/{dataset_id}/assets:
+#    get:
+#      summary: Retrieve links for downloading a Dataset
+#      tags:
+#        - collection
+#      description: |
+#        Generate Presigned S3 URLs to download the files associated with the `dataset_id` in the Collection specified
+#        by the `collection_id`. Pre-signed S3 URLs have a life span of up to 1 hour. You should generate a pre-signed
+#        URL immediately before each download to ensure it is current.
+#      operationId: backend.curation.api.v1.curation.collections.collection_id.datasets.dataset_id.assets.get
+#      parameters:
+#        - $ref: "#/components/parameters/path_collection_id"
+#        - $ref: "#/components/parameters/path_dataset_id"
+#      responses:
+#        "200":
+#          description: OK
+#          content:
+#            application/json:
+#              schema:
+#                $ref: "#/components/schemas/asset_response"
+#        "202":
+#          description: Unable to retrieve one or more asset response fields.
+#          content:
+#            application/json:
+#              schema:
+#                $ref: "#/components/schemas/asset_response"
+#        "400":
+#          $ref: "#/components/responses/400"
+#        "404":
+#          $ref: "#/components/responses/404"
+
+  /v1/datasets:
     get:
-      summary: Retrieve links for downloading a Dataset
+      summary: List Cell Census previews of all Datasets
       tags:
-        - collection
+        - dataset
       description: |
-        Generate Presigned S3 URLs to download the files associated with the `dataset_id`. 
-        Pre-signed S3 URLs have a life span of up to 1 hour. You should
-        generate a pre-signed URL immediately before each download to ensure it is current.
-      operationId: backend.curation.api.v1.curation.collections.collection_id.datasets.dataset_id.assets.get
+        List previews of all Datasets.
+      operationId: backend.curation.api.v1.curation.datasets.actions.get
+      parameters:
+        - name: query_dataset_fields
+          description: |
+            List of Dataset fields to return, by name. See top-level attributes of `GET /v1/datasets/{dataset_id}`
```

Comment:
Despite the "hacky" snark in 64c7ea7a2445661f8ef8ce35d60c40f8a7657d39, it would make sense to have a symmetrical query function for the list of available fields?